src/ParametricOptInterface.jl

```diff
@@ -199,6 +208,10 @@ function MOI.get(model::ParametricOptimizer, ::MOI.NumberOfVariables)
     return length(model.parameters) + length(model.variables)
 end
 
+function MOI.get(model::ParametricOptimizer, attr::MOI.ListOfConstraints)
+    return MOI.get(model.optimizer, attr)
+end
```

Comment:
This method is a little more complicated than that. This happens because some Constraints may change type. 
Ex when you add `p * x` in an expression it enters as an SQF but since `p` is a parameter it transforms into a number and becomes a `SAF` in the inner optimizer

Reply:
I added this because otherwise, the model print fails. I can change it accordingly but not having it causes failures